tests/fixtures/no-internal/workspace-pkg-1/tsconfig.test.json

```diff
@@ -6,5 +6,5 @@
     "esModuleInterop": true,
     "lib": ["es2015", "es2017", "esnext"],
   },
-  "include": ["*.ts"]
+  "include": ["*.ts", "node_modules/@itwin/test-pkg-3/index.d.ts"],
```

Comment:
can you explain why this is necessary?

Reply:
well, you should probably have a dep on test-pkg-3 in your deps right?

Reply:
Which dependencies do you mean? None of the test fixtures have the fake dependencies actually listed in their package.jsons, so I don't think it's needed

Side note I just found that it works if I include `node_modules/@itwin/test-pkg-3/*.ts` so I think something is going wrong with `**` recursive globbing